### PR TITLE
[node]-22-bookworm release changes

### PR DESCRIPTION
**Pull request by gauravsaini04**
**Templates names**:
 
 * javascript-node, javascript-node-mongo, javascript-node-postgres
 
 **Description**:
 
 This PR makes the following changes :
 
 * `devcontainer-template.json` files for all 3 templates have been updated to now have `22-bookworm` version of node to be latest
 * In response to issue #[1015](https://github.com/devcontainers/images/issues/1015)
 
 **Checklist**:
 
 * [x]   Checked that applied changes work as expected

### CODE REVIEW

File: src/javascript-node-mongo/devcontainer-template.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
             "type": "string",
```
Comment (samruddhikhandale) [inline src/javascript-node-mongo/devcontainer-template.json:11]: Can you update the default L20 to "22-bookworm" as well?

File: src/javascript-node-mongo/devcontainer-template.json [hunk 2]
```diff
@@ -11,6 +11,7 @@
...
             "description": "Node.js version (use -bullseye variants on local arm64/Apple Silicon):",
             "proposals": [
+				"22-bookworm",
 				"20-bullseye",
```
Comment (samruddhikhandale) [inline src/javascript-node-mongo/devcontainer-template.json:15]: can you add 22-bullseye as well?

File: src/javascript-node-mongo/devcontainer-template.json [hunk 3]
```diff
@@ -11,6 +11,7 @@
             "type": "string",
             "description": "Node.js version (use -bullseye variants on local arm64/Apple Silicon):",
             "proposals": [
+				"22-bookworm",
```
Comment (samruddhikhandale) [inline src/javascript-node-mongo/devcontainer-template.json:14]: Looks like we missed adding `20-bookworm`, can you add as well?

File: src/javascript-node-postgres/devcontainer-template.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
     "id": "javascript-node-postgres",
-    "version": "3.1.0",
+    "version": "3.1.1",
```
Comment (samruddhikhandale) [inline src/javascript-node-postgres/devcontainer-template.json:3]: Can we make same changes (as commented before) for this Template as well?

File: src/javascript-node/devcontainer-template.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
     "id": "javascript-node",
-    "version": "3.1.0",
+    "version": "3.1.1",
```
Comment (samruddhikhandale) [inline src/javascript-node/devcontainer-template.json:3]: Can we make same changes (as commented before) for this Template as well?

File: src/javascript-node-mongo/devcontainer-template.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {
     "id": "javascript-node-mongo",
-    "version": "3.1.0",
+    "version": "3.1.1",
```
Comment (samruddhikhandale) [inline src/javascript-node-mongo/devcontainer-template.json:3]: Can you fix the tests?

Comment (samruddhikhandale) [inline src/javascript-node-mongo/devcontainer-template.json:3]: Let's bump the `minor` version as we are updating the default

**Review by samruddhikhandale — CHANGES_REQUESTED** [review]

File: src/javascript-node-mongo/devcontainer-template.json (hunk 2 above)
Comment (gauravsaini04) [reply src/javascript-node-mongo/devcontainer-template.json:17]: done

File: src/javascript-node-mongo/devcontainer-template.json (hunk 3 above)
Comment (gauravsaini04) [reply src/javascript-node-mongo/devcontainer-template.json:14]: done

File: src/javascript-node-mongo/devcontainer-template.json (hunk 1 above)
Comment (gauravsaini04) [reply src/javascript-node-mongo/devcontainer-template.json:11]: done

File: src/javascript-node-postgres/devcontainer-template.json (hunk 4 above)
Comment (gauravsaini04) [reply src/javascript-node-postgres/devcontainer-template.json]: done

File: src/javascript-node-mongo/devcontainer-template.json (hunk 6 above)
Comment (gauravsaini04) [reply src/javascript-node-mongo/devcontainer-template.json]: done

Comment (gauravsaini04) [reply src/javascript-node-mongo/devcontainer-template.json]: done

File: src/javascript-node/devcontainer-template.json (hunk 5 above)
Comment (gauravsaini04) [reply src/javascript-node/devcontainer-template.json]: done

File: src/javascript-node-mongo/devcontainer-template.json (hunk 6 above)
Comment (gauravsaini04) [reply src/javascript-node-mongo/devcontainer-template.json]: done

**Review by samruddhikhandale — APPROVED** [review]
Looks good, thanks!

**Pull request merged**